app/views/moderator_applications/_form.html.slim

```diff
@@ -0,0 +1,55 @@
+section.card
+  header.card-header
+    strong Moderator Application Form
+
+  .card-body
+    - if application.errors.any?
+      #error_explanation
+        h3 = "#{pluralize(application.errors.count, "error")} prohibited this application from being submitted:"
+        ul
+          - application.errors.messages.each do |message|
+            li = message[1][0]
+
+    p
+      | You will be able to edit your application after you submit it.
+      |  You may receive your response through the email address associated through your account.
+    br
```

Comment:
`br`? Was that *really* the best I could do to fix this formatting?

Reply:
br is fine if it's quick and it works.